src/resources/postcss/components/_navigation.pcss

```diff
@@ -0,0 +1,106 @@
+/* = Navigation
+============================ */
+.tribe-common {
+
+	.tribe-common-c-nav {
+		border-top: 1px solid var(--color-border-default);
+		padding-top: var(--spacer-6);
+	}
+
+	.tribe-common-c-nav__list {
+		display: flex;
+		list-style: none;
+		width: 100%;
+		flex-wrap: wrap;
+		justify-content: space-between;
+
+	}
+
+	.tribe-common-c-nav__list-item {
+	}
+
+	.tribe-common-c-nav__prev,
+	.tribe-common-c-nav__next {
+		display: flex;
+		justify-content: center;
+		flex-wrap: wrap;
+		align-items: center;
+		color: var(--color-text-secondary);
+		font-family: var(--font-family-base);
+		font-size: var(--font-size-1);
```

Comment:
I came across this in a pass I did, but any place where we're manually defining `font-size` and font-related styles, we should actually not be doing this. On the `tribe-common-c-nav__prev` and `tribe-common-c-nav__next` classes we should be applying, say, `.tribe-common-b1` classes to them. Let's take care of this in another pass.